Configuration/Skimming/test/test_DisTrk_cfg.py

```diff
@@ -0,0 +1,149 @@
+# Auto generated configuration file
+# using: 
+# Revision: 1.19 
+# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
+# with command line options: step1 --filein root://cmsxrootd.fnal.gov//store/mc/Run3Summer21DRPremix/DYToLL_M-4To50_TuneCP5_14TeV-pythia8/GEN-SIM-DIGI-RAW/120X_mcRun3_2021_realistic_v6-v2/80000/03b302a8-8389-406d-945a-16c2f16f065f.root --fileout file:step1_RECO.root --mc --eventcontent AODSIM --runUnscheduled --datatier AODSIM --conditions 120X_mcRun3_2021_realistic_v6 --step RAW2DIGI,L1Reco,RECO,RECOSIM --nThreads 4 --geometry DB:Extended --era Run3 --python_filename test_DisTrksV2_cfg.py --no_exec --customise Configuration/DataProcessing/Utils.addMonitoring -n 100
```

Comment:
Since you claim that "any CMSSW_12 release will be fine", could you please move the test config to a newer release and GT? And in any case, please update the PR description with the correct release you have used for tests (as of now CMSSW_12_0_3_pre2 should be CMSSW_12_0_3_patch2)

Reply:
If you would like this updated it can be done, but I was just following the version used in generating the MC sample. I have updated the PR description to correctly state it is CMSSW_12_0_3_patch2

Reply:
@perrotta Sorry did you want this change made? I just wanted to make sure this wasn't held up waiting for this change.

Reply:
Not necessarily, mine was just a suggestion.
This PR is now waiting for the missing @cms-sw/pdmv-l2 signature: once they will also be happy with it, it can be merged